fulp_modules/ZFulpcode/modules/granters/granters.dm

```diff
@@ -0,0 +1,14 @@
+// Overwrites martial arts granters (granters.dm) to prevent South Star users from getting other martial arts
+// while they are wearing the gloves.
+
+/obj/item/book/granter/martial/on_reading_finished(mob/user)
```

Comment:
whats the point of this instead of just using allow_temp_override = FALSE

Reply:
I tried that, but that only overrides martial arts that are taught as temporary martial arts. Granters like those that give you CQC or Carp teach it to you as a non-temporary martial art.

Reply:
you do realize that not all martial arts come from granters, right?

the warden's krav maga, for example, comes from their gloves, a chef's CQC comes from a skillchip, and so on.

Reply:
you could alternatively just put on the gloves AFTER learning sleeping carp or whatever, no?